### DISCUSSION TITLE
Hide BubbleMenu when selecting text

### DISCUSSION THREAD

**Discussion by SamDuvall** [ueberdosis/tiptap#1522] [Feature Requests]
I love the BubbleMenu, but when I'm selecting text, the BubbleMenu hops all over the place and can sometimes grab the cursor, which causes the selection to go haywire. I'd prefer to not show the BubbleMenu until I'm done selecting text (e.g. my mouse is no longer down). Is something I should handle within my own app or is it something that would be better suited to be a part of `@tiptap/extension-bubble-menu`?

**Comment by robertvanhoesel** [comment]
I've come across the same issue and have implemented an mousedown eventhandler on the editor that adds a `mouse-active` class to the wrapper to hide the tippy bubble menu element whenever the mouse is pressed *on* the editor.

I've also experimented with disabling pointer events on tippy during selection, because occasionally it would trip up and try selecting the bubble menu. I found after some playing around I prefer hiding the menu completely when selecting, similar to medium.com behavior.

I would also really like seeing this logic applied by default in the extension.

**Reply by meatflavourdev** [reply]
Why don't you open and issue and do a pull request?

**Reply by meatflavourdev** [reply]
nm-- someone beat you to it
https://github.com/ueberdosis/tiptap/pull/1692

**Reply by robertvanhoesel** [reply]
@meatflavourdev yeah, also see https://github.com/ueberdosis/tiptap/issues/1493 and the (imho) better PR https://github.com/ueberdosis/tiptap/pull/1556
